src/UI/ledger/LedgerLoginContainer/AddressTable.tsx

```diff
@@ -39,20 +31,30 @@ export interface AddressTablePropsType extends WithClassnameType {
     ledgerModalTableSelectedItemClassName?: string;
     ledgerModalTableNavigationButtonDisabledClassName?: string;
   };
+  customContentComponent?: ReactNode;
+  dataTestId?: string;
+  loading: boolean;
+  onConfirmSelectedAddress: () => void;
+  onGoToNextPage: () => void;
+  onGoToPrevPage: () => void;
+  onSelectAddress: (address: { address: string; index: number } | null) => void;
+  selectedAddress?: string;
+  startIndex: number;
 }
 
 export const AddressTable = ({
-  loading,
   accounts,
-  startIndex,
-  selectedAddress,
-  onGoToPrevPage,
-  onGoToNextPage,
-  onConfirmSelectedAddress,
-  onSelectAddress,
+  addressTableClassNames,
   className = 'dapp-ledger-address-table',
   customContentComponent,
-  addressTableClassNames
+  dataTestId = 'addressTableModal',
```

Comment:
I would only keep addressTableContainer because we can't say it will always be included in a modal